packages/apps/frontera/src/routes/organizations/src/components/Columns/invoices/Cells/Contract/ContractCell.tsx

```diff
@@ -20,26 +20,34 @@ export const ContractCell = observer(
     const organization = store.organizations?.value?.get(organizationId)?.value;
     const contract = store.contracts?.value?.get(contractId)?.value;
     const name = contract?.contractName || `${organization?.name}'s contract`;
+    const orgName = organization?.name;
 
     return (
-      <TableCellTooltip
-        hasArrow
-        label={name}
-        align='start'
-        side='bottom'
-        targetRef={itemRef}
-      >
-        <div className='overflow-hidden overflow-ellipsis'>
-          <p className='text-xs text-gray-500'>{organization?.name}</p>
+      <div className='overflow-hidden overflow-ellipsis'>
+        {orgName ? (
           <Link
             ref={itemRef}
             to={`/organization/${organizationId}?tab=account`}
```

Comment:
How the text behaves when it's too long for the column and the org is present (for users it should be most of the cases)?

Reply:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/2dba5749-c908-4967-9d03-0613aa0c646f">

Reply:
awesome 🙌